### PR TITLE
Auth submit

### PR DESCRIPTION
**Pull request by jacques-cadet**
I believe PR #40 should be merged before PR #41

### CODE REVIEW

**Comment by jbonzo** [comment]
Since this PR is based off of #40 I added a WIP label for clarity.

File: podium/talks/templates/talks/submit.html [hunk 1]
```diff
@@ -12,14 +12,28 @@
...
 				    </table>
-				    <input type="submit" value="Submit" />
+				    <button class="btn btn-primary btn-xs" type="submit" value="Submit" >Submit</button>
 			</form>
+
+			{%else%}
+
+				
+				<h4 style="font-size: 30px;text-align:center;color: #005580; text-shadow: 0 0 50px #669996; ">A verified email is required to submit a talk propasal, and you must be logged in. <br> Please <a href="">Register</a> or <a href="{% url 'auth_login'%}">Login</a>.</h4>
+				<br>
```
Comment (jbonzo) [inline podium/talks/templates/talks/submit.html]: In regards to the link tag for Register, where should the link go? Or is the registration page not yet completed?

**Comment by jacques-cadet** [comment]
Not based off of, just should be done first. Why create a link to
registeration when it doesn't exist?

On Aug 17, 2017 9:13 PM, "Ricky Barillas" <notifications@github.com> wrote:

Since this PR is based off of #40
<https://github.com/pyatl/podium-django/pull/40> I added a WIP label for
clarity.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/pyatl/podium-django/pull/41#issuecomment-323234009>,
or mute
the thread

**Comment by jbonzo** [comment]
Well I said "based off" of since the first commit in this PR is the commit for the other PR. Also I'm a little confused by your other message. Was that a reply to my comment regarding the registration link?

**Comment by jacques-cadet** [comment]
Registration update is in PR #40, it includes all necessary code.

**Comment by jbonzo** [comment]
Right. I think we have a slight misunderstanding but I will wait to review this PR anymore until #40 is merged.

**Comment by jacques-cadet** [comment]
Was that a reply to my comment regarding the registration link?

No, It's my reason for making the initial comment, just in question form. 
Nothing personal.

In regards to the link tag for Register, where should the link go? Or is the registration page not yet completed?

Registration update is in PR #40, it includes all necessary code, links and all.
Sorry for the confusion.

**Review by jrrickerson — CHANGES_REQUESTED** [review]
Please update from upstream and resolve the conflicts with your other PR and update your branch, and I should be able to review and merge this shortly.

**Comment by jacques-cadet** [comment]
Closed to correct conflicts.

**Pull request closed without merging**